.github/workflows/image.yml

```diff
@@ -90,6 +90,7 @@ jobs:
           tags: |
             type=sha,prefix=
             type=sha,prefix=,format=long
+            type=sha,prefix=,format=long,suffix=-${{ matrix.platform }}
```

Comment:
Should we even push these other's then? seems like w should JUST push the sha w/ platform?